src/main/java/lotto/model/LottoGenerator.java

```diff
@@ -1,97 +1,54 @@
 package lotto.model;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class LottoGenerator {
 	public static final int LOTTO_PRICE = 1000;
```

Comment:
```suggestion
	public static final Money LOTTO_PRICE = new Money(1000);
```

금액을 의미하는 LOTTO_PRICE 상수도 Money 클래스를 사용해볼 수 있을 것 같아요 !

Reply:
반영했습니다 👍